app/src/app/api/v0/inventory/[inventory]/results/route.ts

```diff
@@ -0,0 +1,39 @@
+import UserService from "@/backend/UserService";
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import { NextResponse } from "next/server";
+import { getEmissionResults } from "@/backend/ResultsService";
+import sumBy from "lodash/sumBy";
+
+
+export const GET = apiHandler(async (_req, { session, params: { inventory } }) => {
```

Comment:
Can we create a ticket to write an API test for this endpoint

Reply:
https://openearth.atlassian.net/browse/ON-2404